### PR TITLE
docs(README): Point to hosted docs

### PR DESCRIPTION
**Pull request by anishkny**
Instead of markdown `docs/README.md`

### CODE REVIEW

**Review by dgozman — COMMENTED** [review]
Thank you for the PR! There are ~10 more links in README.md, and we should update either all of them or none.

**Comment by anishkny** [comment]
@dgozman sound fair. I pretty much replaced, e.g.:

```diff
- docs/FOO.md[#BAR]
+ https://playwright.dev/#?path=docs/FOO.md[&q=BAR]
```

LMK if this works!

File: README.md [hunk 1]
```diff
@@ -134,8 +134,8 @@ const { webkit } = require('playwright');
 
 ## Resources
 
-* [Documentation](docs/README.md)
-* [API reference](docs/api.md)
-* [Example recipes](docs/examples/README.md)
+* [Documentation](https://playwright.dev/)
```
Comment (dgozman) [inline README.md:137]: This should be `https://playwright.dev/#?path=docs/README.md`

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [![npm version](https://img.shields.io/npm/v/playwright.svg?style=flat)](https://www.npmjs.com/package/playwright) [![Join Slack](https://img.shields.io/badge/join-slack-infomational)](https://join.slack.com/t/playwright/shared_invite/enQtOTEyMTUxMzgxMjIwLThjMDUxZmIyNTRiMTJjNjIyMzdmZDA3MTQxZWUwZTFjZjQwNGYxZGM5MzRmNzZlMWI5ZWUyOTkzMjE5Njg1NDg) <!-- GEN:chromium-version-badge -->[![Chromium version](https://img.shields.io/badge/chromium-86.0.4211.0-blue.svg?logo=google-chrome)](https://www.chromium.org/Home)<!-- GEN:stop --> <!-- GEN:firefox-version-badge -->[![Firefox version](https://img.shields.io/badge/firefox-78.0b5-blue.svg?logo=mozilla-firefox)](https://www.mozilla.org/en-US/firefox/new/)<!-- GEN:stop --> [![WebKit version](https://img.shields.io/badge/webkit-14.0-blue.svg?logo=safari)](https://webkit.org/)
 
-##### [Docs](docs/README.md) | [API reference](docs/api.md) | [Changelog](https://github.com/microsoft/playwright/releases)
+##### [Docs](https://playwright.dev) | [API reference](https://playwright.dev/#?path=docs/api.md) | [Changelog](https://github.com/microsoft/playwright/releases)
```
Comment (dgozman) [inline README.md:5]: This should be `https://playwright.dev/#?path=docs/README.md`

File: README.md (hunk 1 above)
Comment (anishkny) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (anishkny) [reply README.md]: Done

**Review by dgozman — APPROVED** [review]
Looks great!

**Pull request merged**